### ISSUE TITLE
New accuracy failures compared with 0709 baseline

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#602]
### 🐛 Describe the bug

Nightly https://github.com/intel/torch-xpu-ops/actions/runs/9957414400/job/27509359347
xpu  train jx_nest_base
E0716 18:31:15.181000 140283238451008 torch/_dynamo/utils.py:1541] RMSE (res-fp64): 0.00285, (ref-fp64): 0.00003 and shape=torch.Size([512]). res.dtype: torch.float16, multiplier: 3.000000, tol: 0.010000
E0716 18:31:15.181000 140283238451008 torch/_dynamo/utils.py:1433] Accuracy failed for key name norm.bias.grad
fail_accuracy

xpu  train convnext_base
E0716 19:50:58.170000 140456569096000 torch/_dynamo/utils.py:1541] RMSE (res-fp64): 0.02741, (ref-fp64): 0.01008 and shape=torch.Size([128, 1, 7, 7]). res.dtype: torch.float16, multiplier: 2.000000, tol: 0.010000
E0716 19:50:58.170000 140456569096000 torch/_dynamo/utils.py:1433] Accuracy failed for key name stages.0.blocks.0.conv_dw.weight.grad
fail_accuracy

### Versions
PVC 1100, driver 803.61, bundle 0.5.1
Last known good for jx_nest_base
pytorch: https://github.com/pytorch/pytorch/commit/32e74ed
torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/1dcaf3e 
Last known good for convnext_base
pytorch: https://github.com/pytorch/pytorch/commit/dadc0ed
torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/45e55a3

**Comment by retonym** [comment]
could pass with `_adaptive_avg_pool2d_backward` fallback to cpu
